addon/content/intro-panel.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Tracking Protection Messaging Study Intro Panel</title>
+    <link
+      rel="stylesheet"
+      href="resource://tracking-protection-messaging/content/styles/intro-panel.css" >
+  </head>
+  <body>
+    <div id="tracking-protection-study-intro-panel-box">
+      <header class="tracking-protection-study-header">
+        <img class="tracking-protection-study-hero" src="https://raw.githubusercontent.com/mozilla/blok/master/src/img/blok-96.png" />
```

Comment:
I know this is WIP, but should we be loading this from the new "addon/content/shield-icon.png" asset added below instead of 3rd party?

Reply:
Excellent point; a consequence of prototyping in codepen first and not changing the URL. Updated, thanks.